pkg/apis/pipeline/v1beta1/pipeline_types_test.go

```diff
@@ -223,7 +223,7 @@ func TestPipelineTask_ValidateRegularTask_Success(t *testing.T) {
 	tests := []struct {
 		name            string
 		tasks           PipelineTask
-		enableAPIFields bool
+		enableAPIFields string
```

Comment:
Since this was bool earlier, the default was `false`. What is the default now? Is it an empty string? May be explicitly set `enableAPIFields` for all tests to the desired value instead of just setting to `beta` where needed?

Reply:
Thanks that makes sense! I just change the condition to if not set just do not update the ctx configMap value, this might not require maintainers to edit every time even if it is just ctx.background. WDYT?

Reply:
It seems that the reason for this change here is that resolver was an alpha field and this was previously used for setting alpha fields but now it has become beta. I think just in case that we are having more alpha features being used in this case, leaving it as `enableAPIFields` might leave more future room instead of changing back to enableAlphaFields or WithContext again?